src/webhook_app.py

```diff
@@ -3,46 +3,27 @@
 """
 import flask
 import flask.logging
-from flask import Flask, Response, request, render_template
+from   flask import Flask, Response, request
```

Comment:
Errant whitespace change

```suggestion
from flask import Flask, Response, request
```

Reply:
Errant like a Knight, just keepin it clean! :D